privacy_evaluator/attacks/membership_inference/membership_inference_analysis.py

```diff
@@ -0,0 +1,129 @@
+from dataclasses import dataclass
+from typing import Iterable, Type
+import numpy as np
+from . import MembershipInferenceAttack
+from .data_structures.attack_input_data import AttackInputData
+from ...classifiers import Classifier
+from .data_structures.slicing import Slicing
+from .data_structures.slicing import Slice
+from sklearn import metrics
+from textwrap import indent
+
+
+@dataclass
+class MembershipInferenceAttackAnalysisSliceResult:
```

Comment:
Maybe it would make sense to put this `@datacass` under the `data_structure` folder.

Reply:
I think it's a matter of preference. This particular data class is only used and will only ever be used by the `MembershipInferenceAttackAnalysis` class so for me it makes more sense to keep them together. Maybe we can wait until the sprint review and then see what our stakeholders have to say.